### PR TITLE
EDU-2367-EN-Create-API-overview

### PR DESCRIPTION
**Pull request by ghost**
Related issue: [EDU-2367](https://aziontech.atlassian.net/browse/EDU-2367)

Changes:

* Introducing the `Getting started with Azion API`.

[EDU-2367]: https://aziontech.atlassian.net/browse/EDU-2367?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+This documentation will guide you on the first steps of using the **Azion API**.
+
+## API overview
+
+The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. The complete Azion API collection is on Postman.
+
+<Button href="https://api.azion.com/" text="Go to Azion API documentation"></Button>
+
+Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`) so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx:19]:
```suggestion
Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`) so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on [Real-Time Manager (RTM)](https://manager.azion.com).
```

Comment (hannah-sk) [reply src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx:19]: there's no _the_ before RTM.

File: src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+:::tip
+Access the [API introduction](https://api.azion.com/#intro) documentation to find reference information about the API.
+:::
+
+## Prerequisites
+
+To run **Azion API** requests you need:
+
+- [Real-Time Manager (RTM)](/en/documentation/products/accounts/creating-account/) account.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx:29]:
```suggestion
- A [Real-Time Manager (RTM)](/en/documentation/products/accounts/creating-account/) account.
```

**Review by hannah-sk — CHANGES_REQUESTED** [review]
2 small corrections!

**Review by hannah-sk — APPROVED** [review]

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+## Prerequisites
+
+To run **Azion API** requests you need:
+
+- A [Real-Time Manager (RTM)](/en/documentation/products/accounts/creating-account/) account.
+- A [personal token](/en/documentation/products/accounts/personal-tokens/).
+
+## Make API requests
+ 
+Azion API documentation offers code examples to make requests with the terminal, using the curl command, or to automate them with programming languages.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx:34]:
```suggestion
The [Azion API introduction](https://api.azion.com/) offers code examples to make requests with the terminal, using the cURL command, or to automate them with programming languages. 
```

File: src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- A [Real-Time Manager (RTM)](/en/documentation/products/accounts/creating-account/) account.
+- A [personal token](/en/documentation/products/accounts/personal-tokens/).
+
+## Make API requests
+ 
+Azion API documentation offers code examples to make requests with the terminal, using the curl command, or to automate them with programming languages. 
+
+Postman offers HTTPS request snippets for many programming languages.
+
+Navigate to the `Example Request` to change from curl to another desired programming language.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx:38]:
```suggestion
Navigate to the `Example Request` to change from curl to another desired programming language.

You may also clone the collection and run it in Postman by clicking the **Run in Postman** button.
```

**Pull request merged**